Server - Add note about useful plugins on server (#7700)
qgis/QGIS-Documentation: docs/server_manual/plugins.rst

 unzip master.zip
 mv qgis-helloserver-master HelloServer

.. warning::

>     According its `documentation <https://plugins.qgis.org/plugins/HelloServer/>`_, this plugin is designed for

Reviewer: ```suggestion
    According to its `description <https://plugins.qgis.org/plugins/HelloServer/>`_, HelloServer plugin is designed for
```
